src/frontend/src/handler/create_mv.rs

```diff
@@ -107,18 +114,26 @@ pub async fn handle_create_mv(
 ) -> Result<PgResponse> {
     let session = context.session_ctx.clone();
 
+    let is_independent_compaction_group;
     let (table, graph) = {
         {
             let catalog_reader = session.env().catalog_reader().read_guard();
             let (schema_name, table_name) = Binder::resolve_table_name(name.clone())?;
+            is_independent_compaction_group = table_name.ends_with("_al");
```

Comment:
better add some comment, as this is temporary implementation.

Reply:
> better add some comment, as this is temporary implementation.

fixed